### PR TITLE
Update base to debian:stretch

### PR DESCRIPTION
**Pull request by saplla**
- Change base to debian:stretch
- Update node GPG keys

### CODE REVIEW

**Comment by saplla** [comment]
Closes #10.

Tested building rocket.chat:0.63.3 against this. It built, but I haven't got a mongo to test it with. It failed as expected saying it couldn't connect mongo. I couldn't test beyond that.

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
       94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
-      0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
       FD3A5288F042B6850C66B31F09FE44734EB7990E \
       71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
       DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-      B9AE9905FFD7803F25714661B63B535A4C206CA9 \
       C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+      B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+      56730D5401028683275BD23C23EFEFE93C4CFFFE \
+      77984A986EBC2AA786BC0F66B01FBB92821C587A \
```
Comment (geekgonecrazy) [inline Dockerfile]: Why change the gpg key?  This shouldn't have changed

Comment (saplla) [reply Dockerfile]: Keys at https://github.com/nodejs/node have changed.

**Comment by sampaiodiego** [comment]
Looks like debian is deprecating jessie (archiving at least).. 

I guess this is time to update to `stretch`..

**Comment by geekgonecrazy** [comment]
So that’s why the image build keeps failing.  Can we just build and push this image again?  I think it would update the repo paths if Jessie was pulled again

**Comment by engelgabriel** [comment]
Let's test and merge this for 1.0?

File: Dockerfile (hunk 1 above)
Comment (geekgonecrazy) [reply Dockerfile]: ok also updated to have current list.  Seems to have changed even more.

File: Dockerfile [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
 
+RUN apt-get update \
+	&& apt-get install -y \
+		gpg \
+    && rm -rf /var/lib/apt/lists/*
+
 # gpg: key 4FD08014: public key "Rocket.Chat Buildmaster <buildmaster@rocket.chat>" imported
 RUN gpg --keyserver pool.sks-keyservers.net --recv-keys 0E163286C20D07B9787EBE9FD7F9D0414FD08104
 
 # gpg keys listed at https://github.com/nodejs/node
```
Comment (geekgonecrazy) [inline Dockerfile]:
```suggestion
# gpg keys listed at https://github.com/nodejs/node#release-keys
```

**Comment by geekgonecrazy** [comment]
Thanks @saplla 🙏

**Pull request merged**